app/components/views/LNPage/ChannelsTab/hooks.js

```diff
@@ -123,7 +128,22 @@ export function useChannelsTab() {
     } else {
       nodePubKey = node;
     }
-    if (!secp256k1.publicKeyVerify(Buffer.from(nodePubKey, "hex"))) {
+
+    let isPubKeyValid;
+    try {
+      if (!isHexValid(nodePubKey)) {
+        throw new Error("pubkey is not a valid hex");
+      }
+      const pubKeyBuffer = new Uint8Array(Buffer.from(nodePubKey, "hex"));
```

Comment:
Seems this needs to drop the `new Uint8Array` bit, otherwise the UI throws an exception when copy pasting the address for my test node https://testnet-dcrln-01.matheusd.com/ (address is at the bottom of the page, in the footer)

Reply:
The tests did not fail, and I left it by mistake :( 
Thanks for catching it.